themes/default/content/docs/esc/other-integrations/docker.md

```diff
@@ -0,0 +1,126 @@
+---
+title_tag: Integrate with Docker | Pulumi ESC
+title: Docker
+h1: "Pulumi ESC: Integrate with Docker"
+meta_desc: This page provides an overview on how to use Pulumi ESC with Docker.
+weight: 2
+menu:
+  pulumiesc:
+    parent: esc-other-integrations
+    identifier: esc-other-integrations-docker
+---
+
+## Overview
+
+Pulumi ESC integrates with [Docker](https://www.docker.com/) to help developers automatically manage configuration and secrets while running `docker` commands.
+
+## Prerequisites
+
+To complete the steps in this tutorial, you will need to install the following prerequisites:
+
+- the [Pulumi ESC CLI](/docs/esc-cli/)
+- the [Docker CLI](https://www.docker.com/) and shell integration
+
+## Manage environment variables for Docker containers
+
+### Create an ESC environment with environment variables
+
+ESC integrates with `docker` by exporting environment variables from an opened environment. Before you can configure `docker`, you'll need to create an environment that exports environment variables. For example, the environment below fetches AWS credentials via OIDC and exports these credentials in environment variables:
```

Comment:
Might want to mention `files` here too

Reply:
Also maybe "ESC integrates with `docker` containers", since we'll later discuss integrating for creds?